mitiq/benchmarks/mirror_qv_circuits.py

```diff
@@ -60,22 +60,16 @@ def generate_mirror_qv_circuit(
     else:
         first_half_depth = int((depth + 1) / 2)
 
-    qv_generated, _ = generate_quantum_volume_circuit(
+    qv_circuit, _ = generate_quantum_volume_circuit(
         num_qubits, first_half_depth, seed=seed, decompose=decompose
     )
-    qv_half_circ, _ = convert_to_mitiq(qv_generated)
+    qv_circuit = cast(cirq.Circuit, qv_circuit)
 
-    mirror_half_circ = cirq.Circuit()
-    qv_half_ops = list(qv_half_circ.all_operations())
-    for i in range(len(qv_half_ops))[::-1]:
-        op_inverse = cirq.inverse(qv_half_ops[i])
-        mirror_half_circ.append(op_inverse, strategy=cirq.InsertStrategy.NEW)
-
-    circ = qv_half_circ + mirror_half_circ
+    mirror_qv_circuit = qv_circuit + cirq.inverse(qv_circuit)
```

Comment:
Thanks for catching this @natestemen! 

I don't remember why I added L68-74, but commit history shows it's due to `mypy`? I need to look into item 3 in the PR description.

Reply:
No worries! Sometimes things get missed, or maybe there was actually a really good reason to have it that way that no longer is needed!